### PR TITLE
Ref Task 6247: Create search screen UI

### PR DESCRIPTION
**Pull request by trieuthan92pro**
Link redmine: https://edu-redmine.framgia.vn/issues/6247
screenshot from 2019-02-20 17-58-56
screenshot from 2019-02-20 17-59-15
screenshot from 2019-02-20 17-59-35

### CODE REVIEW

**Comment by trieuthan92pro** [comment]
Ready

File: app/src/main/res/drawable/button_background_history_search.xml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+        android:width="@dimen/width_1"
+        android:color="#cbcbcb" />
+    <corners android:radius="@dimen/radius_4" />
+
+    <padding
+        android:bottom="@dimen/width_1"
+        android:left="@dimen/width_1"
+        android:right="@dimen/width_1"
+        android:top="@dimen/width_1" />
+</shape>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/drawable/button_background_history_search.xml]: need an endline

File: app/src/main/res/layout/activity_search.xml [hunk 2]
```diff
@@ -18,14 +18,15 @@
...
             android:layout_width="wrap_content"
             android:layout_height="@dimen/height_24"
             android:layout_gravity="start"
             android:adjustViewBounds="true"
-            android:src="@drawable/icon_left_arrow" />
+            android:src="@drawable/ic_back_24dp" />
 
         <SearchView
-            android:id="@+id/searchView"
+            android:id="@+id/search_siew"
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/activity_search.xml]:
```suggestion
            android:id="@+id/search_view"
```

File: app/src/main/res/layout/history_search_item.xml [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+        android:id="@+id/text_history_search"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_margin="@dimen/margin_8"
+        android:background="@drawable/button_background_history_search"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toTopOf="parent" />
+</android.support.constraint.ConstraintLayout>
```
Comment (NguyenMinhTuanB) [inline app/src/main/res/layout/history_search_item.xml]: need an endline

**Comment by trieuthan92pro** [comment]
Ready

**Review by NguyenMinhTuanB — APPROVED** [review]

**Pull request merged**